### PR TITLE
Added SASS to the list!

### PR DESCRIPTION
**Pull request by jeroenvisser101**
You gotta show it for them to know it, right?

### CODE REVIEW

**Comment by jeroenvisser101** [comment]
(I see I removed the 'oxford comma', should I add it?)

File: app/views/home/index.haml [hunk 1]
```diff
@@ -5,7 +5,7 @@
   %li.step.pitch
     .tagline
       %h2
-        Review your JavaScript, CoffeeScript, and Ruby code
+        Review your JavaScript, CoffeeScript, SASS and Ruby code
```
Comment (salbertson) [inline app/views/home/index.haml]: SCSS?

Comment (croaky) [inline app/views/home/index.haml]: Yeah, should be SCSS.

**Comment by croaky** [comment]
@jeroenvisser101 I just edited that commit to be SCSS and merged as 38dca18. Thanks for catching this!

**Comment by jeroenvisser101** [comment]
Cool! Thanks :)

**Pull request closed without merging**